ONSdigital/eq-author
# Bug - Cypress failures

### What is the context of this PR?
1. Fix issue with withEntityEditor overwriting changes in progress when it received data from network.
1. Fix issue with RTE sometimes not being blurring and hence causing tests to blow up as titles haven't been updated. 
1. Upgrade cypress

### How to review 
1. Ensure that you don't lose data if you edit a field whilst an on going network request is in flight.
1. Ensure you can run cypress locally.
